exercises/practice/resistor-color-trio/lib/resistor_color_trio.ex

```diff
@@ -0,0 +1,8 @@
+defmodule ResistorColorTrio do
+  @doc """
+  Calculate the resistance value in ohm or kiloohm from resistor colors
+  """
+  @spec label(colors :: [atom]) :: {integer, atom}
```

Comment:
We could be a bit more specific:
```suggestion
  @spec label(colors :: [atom]) :: {integer, :ohms | :kiloohms}
```
I mean it doesn't matter, but if it were a real library it would be more useful to know that the atom is not any atom, but a unit.

Reply:
Yes, it's nicer, thank you.